src/libs/antares/study/header.cpp

```diff
@@ -54,7 +56,10 @@ void StudyHeader::reset()
     dateCreated = ::time(nullptr);
     dateLastSave = dateCreated;
     // Author
-    author = STUDYHEADER_DEFAULT_AUTHOR;
+    String username;
```

Comment:
If we wanted, we could try to get rid of `Yuni::String` s (at least **username**, **author** and **editor**), and replace them with `std::string`s.
Unfortunately, it may force us to make changes in **inifile.*** source files.
In case we want/could to do that, we can't replace `System::Username(username);` with a piece of code found [here](https://github.com/tanpero/getUserName).